Automattic/woocommerce-payments
# Add customer link in disputes list

Fixes #3698 

#### Changes proposed in this Pull Request









Adds a link to the customer (if available) in the disputes list.  

#### Testing instructions





* Open Disputes page
* Double check the user name is clickable if the `order.customer_url` is exposed for the dispute
* Double check the user name is not clickable, but still links to the dispute page, if `order.customer_link` is not exposed

-------------------


**Post merge**
